.github/workflows/build-timestamped-master.yml

```diff
@@ -34,12 +34,5 @@ jobs:
           publishPAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
         run: |
           ./gradlew publish --scan --no-daemon
-      - name: Archive Error Log
```

Comment:
Lets use the latest checkout and setup-java actions

Reply:
Change added in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/6/commits/51453f0451418aae24c24df75255e7bb9d1c3800